pkg/sfu/videolayerselector/simulcast.go

```diff
@@ -0,0 +1,143 @@
+package videolayerselector
```

Comment:
By doing this kind of design, if we have VP9 or AV1 do simulcast, we can mix and match and use this selector with VP9/AV1 also.